code/game/machinery/computer/security.dm

```diff
@@ -657,6 +685,16 @@ What a mess.*/
 	updateUsrDialog()
 	return
 
+/obj/machinery/computer/secure_data/proc/get_photo(var/mob/user)
+	if (istype(user.get_active_hand(), /obj/item/weapon/photo))
+		var/obj/item/weapon/photo/photo = user.get_active_hand()
+		return photo.img
+	if (istype(user, /mob/living/silicon))

```

Comment:
I think having two istype checks here is bad, when you can get away with just using one.